Makefile

```
@@ -201,6 +201,20 @@ test-unit-coverage: test-unit gocov
 	$(GOCOV) convert build/coverage/coverprofile > build/coverage/coverprofile.json
 	$(GOCOV) report build/coverage/coverprofile.json
 
+.phony: test-unit-ginkgo
```

Nit: Keep the same style as the other targets.
```suggestion
.PHONY: test-unit-ginkgo
```

---

Good catch. Thank you.